zilencer/management/commands/populate_db.py

```diff
@@ -338,6 +344,20 @@ def handle(self, **options: Any) -> None:
                     "errors": {"description": "For errors", "invite_only": False},
                     "sales": {"description": "For sales discussion", "invite_only": False}
                 }  # type: Dict[Text, Dict[Text, Any]]
+
+                for i in range(options['extra_streams']):
+                    # Since a stream with name "Extra Stream 3" could show up
+                    # after "Extra Stream 29", pad the numbers with 0s.
+                    maximum_digits = len(str(options['extra_streams'] - 1))
```

Comment:
You can compute this outside of the loop.

Reply:
OK, that makes sense. I created #7973 to address that (it just moves it outside of the loop). Thanks!

Reply:
@showell OK, that makes sense. I created #7973 to address that (it just moves the `maximum_digits` calculation outside of the loop). Thanks!